public/app/features/dashboard-scene/scene/DashboardSceneRenderer.tsx

```diff
@@ -24,6 +24,7 @@ export function DashboardSceneRenderer({ model }: SceneComponentProps<DashboardS
   const bodyToRender = model.getBodyToRender();
   const navModel = getNavModel(navIndex, 'dashboards/browse');
   const isHomePage = !meta.url && !meta.slug && !meta.isNew && !meta.isSnapshot;
+  const isDashboardControlsHidden = !controls?.hasControls();
```

Comment:
If you rename this to `hasControls` then there is no need to have three `!` inverts here (and in the markup) as everwhere isDashboardControlsHidden is used it's used with an inverter :)